app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -142,6 +145,14 @@ class BrowserActivity : DuckDuckGoActivity(), CoroutineScope by MainScope() {
         }
     }
 
+    private fun configureEmptyClickListeners() {
```

Comment:
To avoid this since this is a mock view that is not supposed to work anyway, can we make the layout non-clickable? `android:clickable="false"`. I think that should do the trick.

Reply:
Good point, that was clever but I've just tested and it doesn't work.

why? because inside that custom view, in the `onFinishInflate` we are calling:

```
        setOnClickListener {
            super.callOnClick()
        }
```

I'm 99% sure we can remove that part, what do you think?

Reply:
Yeah I think so. I tested it and seems to be fine but if you are not sure, just leave it. Not the end of the world.